client/python/.env

```diff
@@ -0,0 +1,2 @@
+HOST=armada.dev.armadaproject.io
```

Comment:
We should not commit this URL into the repo.  The url is open for all to use.

I think you should just add a placeholder like this: https://stackoverflow.com/a/43664611

Reply:
I forgot to add this to the gitignore... My bad